cfgov/flags/templates/flagadmin/flags/create.html

```diff
@@ -0,0 +1,18 @@
+{% extends "wagtailadmin/base.html" %}
+{% load i18n %}
+{% load gravatar %}
+{% block titletag %}{% trans "Flags" %}{% endblock %}
+
+{% block content %}
+    {% trans "Flags" as flags_str %}
+    {% trans "Add a flag" as add_a_flag_str %}
+
+    {% include "wagtailadmin/shared/header.html" with title=flags_str   %}
+    
```

Comment:
You can see the admin UI above!

I don't think we need flags for whole new content types or sections of the site, because the CMS itself acts already serves that purpose: the CMS will allow us to publish/reveal content. This is more about features.